### ISSUE TITLE
Specific color disappears in the background :(

### ISSUE THREAD

**Issue by fuadsaud** [AlessandroYorba/Monrovia#2]
Thanks for the work, been enjoying the theme so far!

However, this specific colors seems to not be playing well with the background. I'm not sure if this is a problem with my setup (iTerm 2 3 with the Monrovia preset), but I assume this color is wrong. Is this the expected appearance?

![screen shot 2017-01-17 at 17 13 07](https://cloud.githubusercontent.com/assets/1993787/22035756/4996611c-dcd8-11e6-8eae-19810dad73ba.png)

**Comment by AlessandroYorba** [comment]
@fuadsaud good catch. It's either ANSI color #6 or ANSI color #14 that's the culprit. Both colors are darker than the background and used for the LineNr and CursrorLine for any vim users that do not have support for termguicolors. I'll revise those colors later today and issue a new commit.

**Comment by AlessandroYorba** [comment]
@fuadsaud I revised Monrovia.vim and the ANSI color#6. Try the latest commit

**Comment by fuadsaud** [comment]
![screen shot 2017-01-18 at 12 47 27](https://cloud.githubusercontent.com/assets/1993787/22068484/4eb85912-dd7c-11e6-80ae-36b4347d6fcd.png)

Looks great! Thanks!

**Comment by AlessandroYorba** [comment]
@fuadsaud You might see a lighter hue for color #14 in the near future too. Looks good! Thanks dude!

**Comment by fuadsaud** [comment]
Btw, I'm using https://github.com/kien/rainbow_parentheses.vim for lisp development and the parens seem to be displayed in colors that don't match the main palette (they seem too vibrant 😛)

Here's a piece of actual code and a sequence of brackets just to give an idea.
![screen shot 2017-01-18 at 12 52 57](https://cloud.githubusercontent.com/assets/1993787/22068825/67305692-dd7d-11e6-989e-c47f5f4abec4.png)

**Comment by AlessandroYorba** [comment]
I'm not familiar with that plugin but if I had to guess by looking at that screenshot, it would appear that the plugin is targeting a colors from within the 256 color palette. Same with the CursorColumn. You might be able to adjust those by selecting colors from here: https://jonasjacek.github.io/colors/

**Comment by AlessandroYorba** [comment]
This version allows user editable colors https://github.com/luochen1990/rainbow

**Comment by fuadsaud** [comment]
Just hit a similar problem with color #14. This is a screenshot of htop running on iTerm with the latests Monrovia colorscheme.

<img width="1278" alt="screen shot 2017-02-12 at 8 21 18 pm" src="https://cloud.githubusercontent.com/assets/1993787/22866644/0879ce42-f161-11e6-958b-db1be26a09f4.png">

**Comment by AlessandroYorba** [comment]
Yep! ANSI colors [ 5,6,13,14] need revision. I'll push something new very soon.

**Comment by AlessandroYorba** [comment]
@fuadsaud can you do me a favor and see if you like this for #14 [ #282e42 ]. It seems to work in vim as good cursorily background color but I'm wondering if it's legible for terminal use. If not I might have to increase the brightness.

**Comment by fuadsaud** [comment]
![screen shot 2017-02-13 at 10 12 40](https://cloud.githubusercontent.com/assets/1993787/22882985/07c88492-f1d5-11e6-9ac6-ffe2c66d6f39.png)

It's a little better but still too dark to be legible in a low light screen. Does increasing the brightness a little more make it look bad for CursorLine in vim?

**Comment by AlessandroYorba** [comment]
The cursorline result only affects vim users who do not have ```set termguicolors``` enabled. So I guess keeping color #14 dark is only supporting earlier version of vim. The easy fix is to drop support for ANSI all together and just provide a GUI / Truecolor theme with a matching terminal. I'll keep you posted.

**Comment by AlessandroYorba** [comment]
@fuadsaud I've decided to remove the cursorline background from the ANSI version for Sidonia and Monrovia for the time being. Both themes are experiments with 24bit color. It was my intention that users get the full experience by making use of ```set termguicolors``` within Vim along with a matching terminal theme.  So by have cursorline make use of a broken terminal color kinda defeats the purpose. Expect a new terminal theme shortly and an updated Vim file. If there is an alternative/solution that I'm missing, just holla :octocat:

**Comment by AlessandroYorba** [comment]
@fuadsaud try the latest now, and report back. Thanks

**Comment by fuadsaud** [comment]
It seems the new terminal version prioritizes much colder colors (some pink and brownish shades are gone) - and I'm still not sure how I feel about that, but my first impression is that overall it became a little dull.

I also see you made regular/bright colors correspondents all the same, I believe for consistency.

htop displays fine now, and I don't see any other problems with colors blending in the background. Thanks a lot!

**Comment by fuadsaud** [comment]
okay, I played a little bit with the new colorscheme and rewrote both my prompts (bash and zsh) to play in harmony with them, and I think I like it. I miss the presence of a brownish color though, and I was thinking one of blue, green, yellow or bright white color positions could be substituted for a shade like that. What do you think?

**Comment by fuadsaud** [comment]
Also, I was wondering about your interest in building a theme for vim-airline (https://github.com/vim-airline/vim-airline-themes). I've been using papercolor (https://github.com/vim-airline/vim-airline/wiki/Screenshots#papercolor) with monrovia and it looks pretty good, but it's not optimal, I believe.

**Comment by AlessandroYorba** [comment]
@fuadsaud I miss the tan color ```#b8937c``` as well. The only color in the terminal that technically is not being used by Vim is the bright pink color located at #5 and #13 ```#cb2656```. And even thou Vim isn't using it, I really should find its back.  Would you be open to me replacing the green #6 and #14 ```#8cb4c0``` with the tan color ```#b8937c". It would look something like this: 
![screen shot 2017-02-25 at 2 18 46 am](https://cloud.githubusercontent.com/assets/11221489/23330368/0f187bfa-fb01-11e6-9409-fecc97e2fba4.png)

**Comment by fuadsaud** [comment]
I'm divided between replacing #6 and #14 and the one labeled as yellow (#3 and #11), since there's already a blueish shade at #2, but either way is fine I think 👍

**Comment by fuadsaud** [comment]
I'll do some testing and get back soon with a better opinion

**Comment by fuadsaud** [comment]
I've been using an old version (`51ea85c`) of monrovia for the last few days and I've actually been enjoying the more colorful feel of ere (mainly within vim). I'm not sure if it has to do with the bright colors column replicating the normal colors set, instead of defining a new set, or if it has to do with the way the colors have been reassigned in the vim colorscheme. I assume this had to do with some of the contrast issues of the previous versions, but I'm not sure. Anyway, I believe it would be good to try to reach a middle ground between the earlier version and the more correct behvior we have now.

**Comment by AlessandroYorba** [comment]
@fuadsaud good point! I'll create a new branch and maybe we can add that pink and tan colors back in.

**Comment by AlessandroYorba** [comment]
@fuadsaud see [monrovia dev branch](https://github.com/AlessandroYorba/Monrovia/tree/dev)

**Comment by AlessandroYorba** [comment]
@fuadsaud new commit to dev branch has a new vim and iterm color theme. Kinda return to form. Desaturated existing colors from Master branch and brought back a few colors from original master push. Let me know your thoughts

**Comment by AlessandroYorba** [comment]
dev build right now: 
![screen shot 2017-03-27 at 12 03 46 am](https://cloud.githubusercontent.com/assets/11221489/24344629/3b279018-1281-11e7-9fdc-310b9dcbbfeb.png)

**Comment by fuadsaud** [comment]
Ok, so, I forked the project and created a branch pointing to `51ea85c28bf2dbb0b0c93fca941093974b9b7900`, which I believe is where you started the dev branch. These are the colors I've been using for a while.

![screen shot 2017-03-30 at 12 46 14 2](https://cloud.githubusercontent.com/assets/1993787/24513173/edd8d4de-1546-11e7-8e1e-014adfddb401.png)

Here's a side by side comparison of current master (left) and the aforementioned revision (right). Both have strengths in my opinion:

master has a cool blue hue that goes well with the grayish background and white text when highlighting keywords. But I'm a little unsure about the3 reddish hues, mainly the brighter one. It looks promising, but it might need some iteration.

Now, I feel like the old version is a more concise set of colors. The yellow shades go great with the dark blue background (thinking a starry night here), with a pink and rusty orange accent to spice it up. While it could certainly be iterated over and some of the colors could be tweaked, I like the big picture and find it pretty pleasant to use, so I would love to keep this scheme around.

In the end, when talking about colors it all comes down to taste and personal opinion 😅 and mine at this point is that master and the old version are too far apart and barely resemble each other, which is an indicator that they could be separate colors schemes.

Let me know what your thoughts are on this and thanks for the time invested in these projects.

**Comment by AlessandroYorba** [comment]
Here's a version that takes the palette from the old build and makes a few subtle changes. As far as background color, we can always add conditionals so that users can choose if they want a blue or a more desaturated one. Once we get the main colors down, I can push it to the master branch.

<img width="1066" alt="screen shot 2017-03-30 at 1 54 35 pm" src="https://cloud.githubusercontent.com/assets/11221489/24525831/b6da949a-1550-11e7-99b2-15ac34faf226.png">
I created a branch for this at: [Monrovia - Nightly](https://github.com/AlessandroYorba/Monrovia/tree/nightly)

**Comment by AlessandroYorba** [comment]
New commit to nightly build:
<img width="1066" alt="screen shot 2017-04-03 at 4 32 21 pm" src="https://cloud.githubusercontent.com/assets/11221489/24635896/60eaae42-188b-11e7-89a7-d8eaff23bc74.png">
